### ISSUE TITLE
Getting language for each variantName?

### ISSUE THREAD

**Issue by kshepherd** [hbz/lobid#422]
Hi there, is there a way to request language codes for each variantName listed for an entity? I'm not sure if this data is even reliably stored, but i thought I would ask just in case!

For example, this record has many variants for Mark Twain in many languages, but I can't programatically determine which is which:
https://lobid.org/gnd/search?q=mark+twain&format=json&filter=type:Person&from=0&size=1

Many thanks!

**Comment by acka47** [comment]
Hi Kim, thanks for your understandable interest in those language tags but unfortunately we do not have any language tags for `variantName` in lobid-gnd. In the RDF from German National Library (which we use for lobid-gnd) language tags are very sparse. As far as I can see they only exist for non-roman scripts, see e.g. http://d-nb.info/gnd/118540238/about/lds which contains the language tags`uz-Cyrl`, `zh-hans`, `zh-Hant`, `und-Hebr`, `ja`. When these few language tags for `variantName` were added in fall of 2019, we decided to ignore them in our data transformation process in order to keep the lobi-gnd API stable (see https://github.com/hbz/lobid-gnd/issues/243#issuecomment-548371516).
